packages/lab/src/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,72 @@
+import { makePrefixer, useControlled } from "@salt-ds/core";
+import { clsx } from "clsx";
+import {
+  type ComponentPropsWithoutRef,
+  forwardRef,
+  type SyntheticEvent,
+  useCallback,
+  useMemo,
+} from "react";
+import { CollapsibleContext } from "./CollapsibleContext";
+
+export interface CollapsibleProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "onToggle"> {
+  /**
+   * Whether the accordion is expanded.
+   */
+  expanded?: boolean;
```

Comment:
Where did we land on this... the Component is called Collapsible but expands, rather than collapses.
Should it be called ExpandiblePanel or something in that case?